listenbrainz_spark/recommendations/recording/create_dataframes.py

```diff
@@ -122,6 +122,12 @@
 #   ]
 
 
+# Dataframe id is made up of two parts.
+# String + UUID
```

Comment:
I find this comment confusing, since there are no UUIDs shown at all. Perhaps explain that it will be added later?

Reply:
I completely agree. I will do it in this PR before merging. Thanks :)